docs/en/writing-running-appium/tutorial/swipe/android-tricks.md

```diff
@@ -0,0 +1,84 @@
+## Android 'UIScrollable' swipe: Tricks and Tips
+
+While Appium does not allow you to use all power of 'UIScrollable'
```

Comment:
This has been changed since 1.18.0@beta. See https://github.com/appium/appium/issues/4733

Reply:
..to use the full power...